util/config.go

```diff
@@ -18,6 +18,7 @@ type Config struct {
 	Username          string
 	Password          string
 	LocalAddress      string
+	RunLevel          string `default:"sysinit"`
```

Comment:
Curious, why use fields here?

I'd like to keep the patterns consistent as much as possible. As in, I'd like for all the config default values to be handled the same way.

Reply:
It was the way I worked out how to add a default.

Which other code has an attribute with a default string value value that I can use an example of how to do it?

Reply:
http tuneout an example of a default.

cmd/goss/goss.go flags & goss/add.go (AddResources)

Reply:
I'm not sure I understand. Do you mean I should add code like:

```
if c.RunLevel == "" {
  c.RunLevel = "sysinit"
}
```

Reply:
@aelsabbahy I've updated this code to no longer use reflection based off the default tag on the struct.
I hope this resolves this objection.